yt/frontends/parthenon/data_structures.py

```diff
@@ -164,10 +160,7 @@ def __init__(
 
         self.geometry = _geom_map[self._handle["Info"].attrs["Coordinates"]]
 
-        if self.geometry is Geometry.CYLINDRICAL:
-            axis_order = ("r", "theta", "z")
-        else:
-            axis_order = None
+        axis_order = None
```

Comment:
You can simply remove this entirely (and remove the corresponding line in the Dataset.__init__() call, line 172 below) and rely on the built in default handling. 

```suggestion
```